x/nftmarket/keeper/nft_listing.go

```diff
@@ -58,6 +59,53 @@ func getTimeKey(prefix string, timestamp time.Time) []byte {
 	return bz
 }
 
+func (k Keeper) SaveNftListing(ctx sdk.Context, listing types.NftListing) {
+	k.SetNftListing(ctx, listing)
+	k.UpdateClass(ctx, listing)
+}
+
+func (k Keeper) UpdateClass(ctx sdk.Context, listing types.NftListing) {
```

Comment:
Function name UpdateClass does not make sense compared to implementation. Can you add any comments on this function?

Reply:
This function checks the listing.state and adds or removes data in the listedClass.
What name is appropriate?
Is updateClass too abstract a name?

Reply:
I think UpdateListedClass would more make sense